pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/kubernetes/KubernetesRuntimeFactory.java

```diff
@@ -181,24 +186,23 @@ public void initialize(WorkerConfig workerConfig, AuthenticationConfig authentic
         this.javaInstanceJarFile = this.pulsarRootDir + "/instances/java-instance.jar";
         this.pythonInstanceFile = this.pulsarRootDir + "/instances/python-instance/python_instance_main.py";
         this.serverCaBytes = workerConfig.getTlsTrustChainBytes();
+        this.manifestCustomizerClassName = factoryConfig.getKubernetesManifestCustomizerClassName();
+        this.manifestCustomizerStaticConfig = factoryConfig.getKubernetesManifestCustomizerConfig();
         try {
             setupClient();
         } catch (Exception e) {
             log.error("Failed to setup client", e);
             throw new RuntimeException(e);
         }
 
-        if (functionAuthProvider.isPresent()) {
-            if (!(functionAuthProvider.get() instanceof KubernetesFunctionAuthProvider)) {
+        this.authProviderClassName = functionAuthProviderClass;
```

Comment:
I don't think we need change this

Reply:
see comment here https://github.com/apache/pulsar/pull/5400/files#r344416501